ui/src/app/applications/components/application-sync-options/application-sync-options.tsx

```diff
@@ -113,6 +113,8 @@ export const ApplicationSyncOptions = (props: ApplicationSyncOptionProps) => (
         ))}
         <div className='small-12' style={optionStyle}>
             {booleanOption('Replace', 'Replace', false, props, false, REPLACE_WARNING)}
+            {selectOption('Prune', 'Prune', 'true', ['true', 'false', 'confirm'], props)}
```

Comment:
Should reorder items so Prune, Prune Last and PrunePropagationPolicy are next to each other.

The UI should also group the boolean options together, then the selectOptions.